admin_manual/maintenance/package_upgrade.rst

```diff
@@ -9,8 +9,11 @@ The best method for keeping ownCloud current on Linux servers is by configuring
 your system to use ownCloud's `Open Build Service`_ repository. Then stay 
 current by using your Linux package manager to install fresh ownCloud packages. 
 After installing upgraded packages you must run a few more steps to complete 
-the 
-upgrade. These are the basic steps to upgrading ownCloud:
+the upgrade. These are the basic steps to upgrading ownCloud:
+
+.. warning:: Make sure that you don't skip a major release when upgrading via repositories.
+   For example you can't upgrade from 8.1.x to 9.0.x directly as you would skip the 8.2.x
+   major release. Scroll down to Upgrading Across Skipped Releases for more info.
```

Comment:
The issue here is that you can't use anchors ``#upgrading-across-skipped-releases`` in :doc: links. So we would need to use absolute URLs like:

https://doc.owncloud.org/server/9.2/admin_manual/maintenance/package_upgrade.html#upgrading-across-skipped-releases

which needs updating of the version after each release.